### PR TITLE
dev/core#343 Fix long address lines overflowing label

### PR DESCRIPTION
**Pull request by andrewpthompson**
Overview
----------------------------------------
When a name or address is very long (long enough for the text to wrap onto a new line) it overflows the label. This can be seen by comparing the PDF output by CiviCRM with an official Avery template. This fixes that by modifying ```CRM_Utils_PDF_Label::generateLabel()``` so that the width that it uses has the left & right padding subtracted from the label's width.

Before
----------------------------------------
Long lines can overflow the label boundary.

After
----------------------------------------
Long lines wrap within the label boundary with correct padding applied.

https://lab.civicrm.org/dev/core/issues/343

### CODE REVIEW

**Comment by aydun** [comment]
Tested, works as described.  Good to merge.

File: CRM/Utils/PDF/Label.php [hunk 1]
```diff
@@ -185,7 +185,7 @@ public function LabelSetFormat(&$format, $unit) {
    */
   public function generateLabel($text) {
     $args = array(
-      'w' => $this->width,
+      'w' => $this->width - 2 * $this->paddingLeft,
```
Comment (MegaphoneJon) [inline CRM/Utils/PDF/Label.php]: 2 is a magic number here.  Could you please add a comment explaining why this change works?  Thanks!

**Comment by andrewpthompson** [comment]
Thanks @MegaphoneJon and @aydun for the feedback. I've added a comment in response to Jon's review comment (good point).

**Comment by eileenmcnaughton** [comment]
I feel review here is adequate - merging
